### PR TITLE
Only expose RavenDB 5 on localhost

### PR DESCRIPTION
**Pull request by andreasohlund**
Fixes https://github.com/Particular/ServiceControl/issues/3264

### CODE REVIEW

File: src/ServiceControl.Audit.Persistence.RavenDb5/persistence.manifest [hunk 1]
```diff
@@ -9,11 +9,6 @@
       "Name": "ServiceControl.Audit/DBPath",
       "Mandatory": true
     },
-    {
-      "Name": "ServiceControl.Audit/HostName",
-      "Mandatory": true
-
-    },
```
Comment (mauroservienti) [inline src/ServiceControl.Audit.Persistence.RavenDb5/persistence.manifest:16]: Isn't this also used to set up ServiceControl API?

Comment (andreasohlund) [reply src/ServiceControl.Audit.Persistence.RavenDb5/persistence.manifest:16]: Yes but this only controls what the persisted needs to know, the instance it self will still have and use the setting

**Review by mikeminutillo — APPROVED** [review]
As a consequence of this, the database will not be available outside of the local server, even in maintenance mode. I don't think that's an issue but it should probably be mentioned in the documentation somwhere.

**Comment by andreasohlund** [comment]
@mikeminutillo docs already states that localhost must be used

https://docs.particular.net/servicecontrol/audit-instances/creating-config-file#troubleshooting-ravendb-5

where

3.5 points out that hostname also works

https://docs.particular.net/servicecontrol/audit-instances/creating-config-file#troubleshooting-ravendb-3-5

**Pull request merged**